sdk/appconfiguration/app-configuration/test/public/tracing.spec.ts

```diff
@@ -45,11 +45,13 @@ describe("supports tracing", () => {
         "AppConfigurationClient.deleteConfigurationSetting",
       ],
     );
+    */
     try {
       await client.setReadOnly({ key: key }, false);
       await client.deleteConfigurationSetting({ key: key });
+      assert.ok(true);
```

Comment:
huh that's unexpected - why do you need this?

Reply:
We don't 😅 It was added in the first commit generating from the dev-tool command and I forget to remove it